### PR TITLE
WA-4061: Implement export transaction data

### PR DESCRIPTION
**Pull request by hailefrollo**
Checklist
Use the ✅ emoji for when it's been done; ⛔ when it still needs work; strikethrough when an item of the checklist does not apply to the changes.

- Is the destination branch correct & intended?  ✅
- Is the correct JIRA ticket referenced?  ✅
- Is the correct API docs/Confluence Page referenced?  ✅
- Have all the AC's in the ticket passed?  ✅
- Have you followed proper naming conventions in the code?  ✅
- Has the code been properly covered with unit tests?  ✅
- Have you regenerated the mock DBs, if the datamodel was modified  ✅
- Have you added migration tests (if applicable)?  ✅
- Have the unit tests passed locally (if applicable)?  ✅
- Have you addressed the code warnings if any?  ✅
- Have you tested the API and made sure the documentation is up-to-date?  ✅
- Have you added documentation for all public methods & types  ✅

### CODE REVIEW

File: Sources/FrolloSDK/Network/API/Requests/APIService+Aggregation.swift [hunk 1]
```diff
@@ -880,4 +880,23 @@ extension APIService {
...
+        requestQueue.async {
+            let url = URL(string: AggregationEndpoint.transactionsExport.path,
+                          relativeTo: self.serverURL)!
+
+
+            self.network.sessionManager.request(url,
+                                                method: .get,
+                                                parameters: parameters,
+                                                encoding: URLEncoding.default,
+                                                headers: nil).validate(statusCode: 200...299).responseData(queue: self.responseQueue) { response in
```
Comment (dipeshfrollo) [inline Sources/FrolloSDK/Network/API/Requests/APIService+Aggregation.swift:895]: I believe you need to support empty response code for Alamofire to properly work like 
`responseData(queue: self.responseQueue, emptyResponseCodes: [204])`

Comment (sincychacko-frollo) [reply Sources/FrolloSDK/Network/API/Requests/APIService+Aggregation.swift:895]: 204, 205 are default values, but it's a different code then we need to add it.

**Review by NeilCossorFrollo — COMMENTED** [review]
Perhaps this is how we normally handle sending a date, but should ExportTransactionFilter have  toDate and fromDate as String ? I know the JSON will actually encode/decode string, but I would have thought the native parameter would be Date? note String?

**Comment by hailefrollo** [comment]
@sincychacko-frollo @dipeshfrollo @NeilCossorFrollo cheers guys, I updated my PR!

**Review by dipeshfrollo — APPROVED** [review]

**Pull request merged**